target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -30,6 +31,7 @@ ramips_setup_interfaces()
 	ubnt,unifi-nanohd)
 		ucidef_set_interface_lan "lan"
 		;;
+	ampedwireless,ally-r1900k|\
```

Comment:
Move blocks due to alphabetic sorting.

Reply:
I'm not following you on this.  Within each port configuration, the device names are already sorted alphabetically.

Reply:
and all the blocks as a whole are sorted based on their first entry ...

Reply:
ahh ok!